src/InnerHit/NestedInnerHit.php

```diff
@@ -145,14 +147,14 @@ public function toArray()
     {
         $out = array_filter(
             [
-                'query' => $this->getQuery()->toArray(),
+                'query' => $this->getQuery() ? $this->getQuery()->toArray() : null,
```

Comment:
This one is mandatory since it will failed with : `Call to a member function toArray() on null`